gadenbuie/xaringanExtra
# New Extension: Broadcast

As suggested in #51, this adds a new feature "broadcast" that can be enabled via `use_broadcast()`.

The demo slides are deployed at https://goofy-darwin-b4e92e.netlify.app/

To test this feature:

1. Open the above slides. 
2. Press <kbd>P</kbd> to enter Presenter mode and follow the instructions there. 
    - Basically, click the the **Broadcast** button.
3. Copy the broadcast link and open in a _different_ browser (i.e. Chrome, Firefox, Safari, or IE but not the same as the first browser) or in a private window.
4. Flip through the slides 😃 

This probably won't work well with proxies or VPNs but should still work in most other normal networking situations.
